### ISSUE TITLE
FMEAN_ productivity variable units

### ISSUE THREAD

**Issue by Viskari** [EDmodel/ED2#121]
Curiosity question. The FMEAN variables for NPP, GPP and respiration terms are per year yet the MsTMIP units for those same variables are for per second.

Why does ED2 look at the annual average for those variables?

**Comment by mpaiao** [comment]
FMEAN are not annual averages, these are just unit conversion to kgC/m2/yr.

**Comment by Viskari** [comment]
No, I understood that, I was asking why are the units converted to
kgC/m2/yr instead of kgC/m2/s?

**Comment by serbinsh** [comment]
All,

I think there are two issues: 1) Clarification of why we use kgC/m2/yr for our timesteps (FMEANS) that are often at 30 minutes.  Personally /yr is hard for me to conceptualize where /s would be clearer; and 2) if we are trying to standardize outputs with the community, as we are with PEcAn, then, for example, MsTMIP uses kgC/m2/s [http://nacp.ornl.gov/MsTMIP_variables.shtml]

@mdietze and David Lebauer (can't find your user in ED2), what are your thoughts?  This is important for unit conversions in PEcAn.

**Comment by serbinsh** [comment]
@Viskari can you please put in the section of model2netcdf code here that shows where we are running unit conversions to clarify the issue?  That is, from what I saw it seems we are running an incorrect conversion if ED2 is outputting kgC/m2/yr instead of kgC/m2/s.  BUT maybe I am getting things mixed up.

**Comment by Viskari** [comment]
Shawn, you are actually slightly getting things mixed up as the current
model2netcdf runs unit conversion from umolC/m2/s to kgC/m2/s. I haven't
added the unit conversion from per year to per second yet as I am trying to
figure out another issue with the output in it.

The current line of code for NPP goes:

out <- conversion( 9, umol2kg_C)  ## umol/m2 s-1 -> kg m-2 s-1

**Comment by mpaiao** [comment]
@Viskari @serbinsh These units are legacy from some discussions we had in the early days of ED-2.1.
I agree it's confusing, I think the output should be always in SI units (this would also reduce unit conversions in the code).

**Comment by serbinsh** [comment]
@mpaiao OK, thanks for the reply.  Is this something we propose within the ED2 community?  I imagine it will require everyone being aware of what is changing and when. We could, for example, propose a list of changes and a timeline starting with the -T- FMEAN variables.
